server/controller/signup.js

```diff
@@ -2,6 +2,20 @@ const bcryptjs = require('bcryptjs');
 const snakeCase = require('snakecase-keys');
 const { users, employeeSkills } = require('../database/models');
 
+exports.checkEmail = async (request, response) => {
+  try {
+    const { email } = request.headers;
+    const emailResult = await users.findOne({ where: { email }, raw: true });
+    if (emailResult) {
+      response.status(400).send('email is already exist !');
```

Comment:
a bit of a grammar mistake. Also, no space before the exclamation mark.
```suggestion
      response.status(400).send('email already exists!');
```